lib/run/export-file.js

```diff
@@ -44,7 +44,9 @@ var _ = require('lodash'),
 module.exports = function (options, done) {
     // parse the path if one is available as string
     var path = _.isString(options.path) && parsePath(options.path),
-        content = _.isPlainObject(options.content) ? JSON.stringify(options.content, 0, 2) : (options.content || E);
+        content = _.isArray(options.content) || _.isPlainObject(options.content) ? // global run variable array exports
+            JSON.stringify(options.content, 0, 2) : // @todo: Fix all associated pieces of logic
```

Comment:
Can you also add a stringify reviver function to use toJSON on objects that has one?